AmericanMedicalAssociation/ama-style-guide-2
# EWL-9971: Update styling for article stub list redesign.

## Ticket(s)
**Do not** submit a Pull Request without a relevant JIRA ticket or Github issue! If you are creating a new pattern or feature, create an issue describing the need for this feature in Github **first** and then link to it below.

**Github Issue**
- [Issue XXX: Issue Title](https://github.com/AmericanMedicalAssociation/AMA-style-guide/issues/XXX)

## JIRA Ticket(s)
- [EWL-9971: Article Stub List Custom Block | Redesign](https://issues.ama-assn.org/browse/EWL-9971)


## Description:
Update templates and series listing view for redesign of article stub list custom blocks.


## To Test:
- import config changes (lando drush @one.local cim -y)
- checkout ama-d8 PR: https://github.com/AmericanMedicalAssociation/ama-d8/pull/3927
- clear caches

- navigate to various category, subcategory, series and topic term pages. confirm all existing stub lust custom blocks match the following zeplins, **especially tablet:**
Desktop: https://zpl.io/KG14ZN8
Tablet: https://zpl.io/PqQvM8A 
Mobile: https://zpl.io/B1lOBpK

- add article stub list custom block
- confirm fields and available paragraphs match the AC in the Jira ticket above

## Visual Regressions

_Please provide the pull request ID below. This will then reference the automated VRT report after it has been generated._

A report is available [here](http://htmlpreview.github.io/?https://github.com/AmericanMedicalAssociation/ama-style-guide-2/blob/visual-regression-testing-artifact/[PULL_REQUEST_ID]/html_report/index.html).

**Before proceeding:** Run visual regression tests locally to ensure new work does not introduce new bugs.

_The visual regression tests always compare against the production version of the style guide. Changes and new work will show up as test failures. Please describe your work so that the report can be reviewed by the team._

If you are creating or updating a pattern be sure you have created or updated the [scenario in `backstop.json`](ama-style-guide-2/styleguide/backstop.json).

_After changes in the report are documented and demonstrate your desired changes, please mark the pull request as `ready for review`.

For more information on visual regression testing review the [How do I run tests?](https://issues.ama-assn.org:8446/confluence/pages/viewpage.action?pageId=23298568) document in Confluence.


## Relevant Screenshots/GIFs
![Screen Shot 2023-03-16 at 3 58 41 PM](https://user-images.githubusercontent.com/67962801/225751751-3d111fca-6907-4e9b-b790-8d7e8ed50702.png)
![Screen Shot 2023-03-16 at 3 58 53 PM](https://user-images.githubusercontent.com/67962801/225751753-80be8158-75aa-4468-9796-d3f02d147219.png)
![Screen Shot 2023-03-16 at 3 59 53 PM](https://user-images.githubusercontent.com/67962801/225751758-8f2d2aae-e1a4-4c35-8696-1e3cbdbcffe4.png)



## Remaining Tasks
Things relevant to here may be updating [The Glossary](https://issues.ama-assn.org:8446/confluence/display/DTD/Glossary?src=contextnavpagetreemode) with the latest naming changes or review of the relative [D8](https://github.com/AmericanMedicalAssociation/ama-d8) work.


## Additional Notes
Anything more to add? Good things to call out here are:
- are there any PRs in this or other repositories that relate to or affect this PR?
- are there any caveats or oddities testers should know about when testing this PR?
- are there any problems you ran into during your work that you'd like to call out?

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)
